doc/manual/installing.rst

```diff
@@ -179,8 +181,8 @@ Installing the host-side software
         $ cd ~/artiq-dev/artiq/doc/manual
         $ make html
 
-Xubuntu 14.04 specific instructions
------------------------------------
+Ubuntu 14.04 specific instructions
```

Comment:
the package sources are mostly same in ubuntu and xubuntu, so the instructions apply to either.